build/sbt

```diff
@@ -20,10 +20,12 @@
 # When creating new tests for Spark SQL Hive, the HADOOP_CLASSPATH must contain the hive jars so
 # that we can run Hive to generate the golden answer.  This is not required for normal development
 # or testing.
-for i in "$HIVE_HOME"/lib/*
-do HADOOP_CLASSPATH="$HADOOP_CLASSPATH:$i"
-done
-export HADOOP_CLASSPATH
+if [ "$HIVE_HOME" ]; then
```

Comment:
Yeah, I only commented because I'm actually somewhat wary of bash-isms and http://unix.stackexchange.com/a/32211/42294 came to mind.